src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -8044,28 +8028,13 @@ components:
       required: true
       schema:
         type: string
-    Depth:
```

Comment:
Instead of deleting, can this be used for `lookupFileByPath` and `lookupSnapshotFileById`?

Reply:
Yes, I'll look into all possible implementations of `Depth`, `AssetId` and others